### PR TITLE
[HOPSWORKS-2777] Improve deletion of large number of executions per job

### PR DESCRIPTION
**Pull request by tkakantousis**

### CODE REVIEW

File: attributes/default.rb [hunk 1]
```diff
@@ -461,10 +461,13 @@
...
 default['hopsworks']['livy_startup_timeout']           = "240"
 
-# Docker job
+# Jobs
 default['hopsworks']['docker-job']['docker_job_mounts_list']    = ""
 default['hopsworks']['docker-job']['docker_job_mounts_allowed'] = "false"
 default['hopsworks']['docker-job']['docker_job_uid_strict'] = "true"
+default['hopsworks']['job']['executions_per_job_limit'] = "10000"
+default['hopsworks']['job']['executions_cleaner_batch_size'] = "1000"
+default['hopsworks']['job']['executions_cleaner_interval_ms'] = "60000"
```
Comment (robzor92) [inline attributes/default.rb:470]: See comment in hopsworks-ee

File: files/default/sql/ddl/updates/2.5.0.sql [hunk 2]
```diff
@@ -8,4 +8,6 @@ ALTER TABLE `hopsworks`.`serving` ADD COLUMN `model_name` varchar(255) COLLATE l
 SET SQL_SAFE_UPDATES = 0;
 UPDATE `hopsworks`.`serving`
 SET `model_name` = (SELECT REPLACE(SUBSTRING(SUBSTRING_INDEX(`model_path`, '/', 5), LENGTH(SUBSTRING_INDEX(`model_path`, '/', 4)) + 1), '/', ''));
-SET SQL_SAFE_UPDATES = 1;
\ No newline at end of file
+SET SQL_SAFE_UPDATES = 1;
+
+ALTER TABLE `hopsworks`.`executions` DROP FOREIGN KEY `FK_347_365`;
```
Comment (robzor92) [inline files/default/sql/ddl/updates/2.5.0.sql:13]: This change also needs to be in initial tables

**Review by robzor92 — CHANGES_REQUESTED** [review]

File: files/default/sql/ddl/updates/2.5.0.sql (hunk 2 above)
Comment (tkakantousis) [reply files/default/sql/ddl/updates/2.5.0.sql:13]: fixed

File: attributes/default.rb (hunk 1 above)
Comment (tkakantousis) [reply attributes/default.rb]: fixed

**Review by robzor92 — APPROVED** [review]

**Review by robzor92 — CHANGES_REQUESTED** [review]
Also the undo for the dml needs to be added

**Comment by tkakantousis** [comment]
> Also the undo for the dml needs to be added

Thanks, fixed.

**Review by robzor92 — APPROVED** [review]

**Pull request merged**